### ISSUE TITLE
ControlGroup Widget HUGE Input Text Box, please help...

### ISSUE THREAD

**Issue by sipi41** [jquery/jquery-mobile#8563]
I'm sorry to bother you but I think I might have discovered a problem with jquery mobile...  I'm trying to reproduce what you show on your examples, please view image  **https://forum.jquery.com/viewFile.do?fileId=14737000007177331&forumGroupId=14737000000003003&portalzaid=-1** but instead I get a huge textbox with a lot of padding... here's the running example https://jsfiddle.net/2qot9tfk/

Some friends at the forum tried to fix this by adding height css property to the textbox as a fix but, it reduces the size not the position of the cursor, and makes it look ugly...

**Comment by deblocker** [comment]
Try this:

.controlgroup-textinput {
  border-width: 1px !important;
  padding: .25em 0 !important;
  border-bottom-style: solid !important;
  -moz-box-shadow: none !important;
  -webkit-box-shadow: none !important;
  box-shadow: none !important;
}
.controlgroup-textinput input {
  text-align: right !important;
  font-weight: 700 !important;
}

**Comment by apsdehal** [comment]
Closing it as not an issue with jquery mobile, more suitable for forums.

**Comment by nexushoratio** [comment]
I think this IS a bug with jquery mobile.  Or at least a documented example.

I was following the example at http://demos.jquerymobile.com/1.4.5/controlgroup/#Textinputs .  This renders into the example shown in the first link.

However, the standalone example I made renders like the jsfiddle above (and the attached image)
![jqm-8563](https://user-images.githubusercontent.com/6006766/36930399-cb5926ee-1e54-11e8-8a79-a7a58d0a0bdf.png)

I have not yet tried any of the work arounds posted, but I expect this should either work out of the box, or the example be updated to provide a working standalone example.

Obviously, the example works on the demo page, but it is using a lot of extra CSS coming from the demo site, so extracting the necessary bits is tricky.

The code below I just loaded as a file: url.  The rendering was the same in both FF 58, Chrome 63, and Chromium 64.

```html
<!DOCTYPE html>
<meta name="viewport" content="width=device-width, height=device-height, initial-scale=1">
<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jquerymobile/1.4.5/jquery.mobile.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script> 
<script src="https://ajax.googleapis.com/ajax/libs/jquerymobile/1.4.5/jquery.mobile.min.js"></script>

<label for="currency-controlgroup">Value</label>
<div data-role="controlgroup" data-type="horizontal">
    <select>
        <option>$</option>
        <option>€</option>
        <option>£</option>
        <option>¥</option>
        <option>₩</option>
        <option>₹</option>
    </select>
    <input id="currency-controlgroup" data-wrapper-class="controlgroup-textinput ui-btn" type="text">
    <button>.00</button>
</div>
<label for="search-control-group">Search</label>
<div data-role="controlgroup" data-type="horizontal">
    <input id="search-control-group" data-wrapper-class="controlgroup-textinput ui-btn" type="text">
    <button>Submit</button>
    <button>Reset</button>
</div>
```

**Comment by nexushoratio** [comment]
Oho!  User error!

When reading the documentation that I linked to, I did not notice the the [View Source] button brought up a pop-up that included an accordion widget.  So I never realized there was a CSS section to open up with the answer right there.

Even though I'd read where it said ``add one css rule'', I still glossed over the CSS section in the source pop-up.
